### DISCUSSION TITLE
Кликабельная ссылка на трек номер клиенту

### DISCUSSION THREAD

**Discussion by Xprinter-dv** [cdek-it/wordpress#38] [Ideas]
Сейчас после отправки накладной, клиенту приходит письмо WP с информацией о трек номере вида
[CDEKDelivery] Номер для отслеживания: 10028291520

Возможно стоит сделать, чтобы в письме можно было кликнуть на трек номер и перейти на сайт СДЭК для отслеживания?
https://www.cdek.ru/ru/tracking?order_id=10028291520

или это можно настроить самостоятельно?

**Comment by vermorag** [comment]
Добрый день. Сейчас трек номер добавляется только как заметка для заказа. Отдельного письма наш плагин не отправляет.
Однако замечу, что при наличии электронной почты клиента, ему от компании СДЭК придет уведомление о принятии заказа на доставку и в нем уже есть все необходимые ссылки.
